package.json

```diff
@@ -31,47 +31,46 @@
   "homepage": "https://github.com/esbb48/Kizuna#readme",
   "dependencies": {
     "babel-polyfill": "^6.9.1",
-    "grommet": "^0.6.10",
+    "extract-text-webpack-plugin": "^v2.0.0-beta.4",
+    "grommet": "^1.0.1",
```

Comment:
hmmm .. 
I didn't find any command.

so I use `npm update` to check which could be update and change it by myself .